src/Eto/Forms/Controls/Control.cs

```diff
@@ -1213,11 +1218,11 @@ public Window ParentWindow
 	public IEnumerable<string> SupportedPlatformCommands => Handler.SupportedPlatformCommands;
 
 	/// <summary>
-	/// Specifies a command to execute for a platform-specific command
+	/// Specifies a command to execute for a Mac platform-specific command
```

Comment:
This would also be used for any other platform that has similar needs, not just Mac.  Even though it is only useful for Mac at this point.